roles/cobbler/templates/kickstarts/cephlab_ubuntu.preseed

```diff
@@ -110,7 +110,9 @@ d-i pkgsel/upgrade select safe-upgrade
 d-i pkgsel/update-policy select none
 
 # Install GRUB to default location.  Required as of Xenial.
+#if $os_version == 'xenial'
```

Comment:
nah.  just file a followup (presuming you have a major or a 'major*100+minor" number you can use)